.ebextensions/env-file-creation.config

```diff
@@ -0,0 +1,41 @@
+# Creates an .env file from AWS SSM Parameter Store
+
+commands:
+  01-create-env:
+    command: "/tmp/create-env.sh"
+
+files:
+  "/tmp/create-env.sh":
+      mode: "000755"
+      content : |
+        #!/bin/bash
+        # Reach into the undocumented container config
+        AWS_REGION='`{"Ref": "AWS::Region"}`'
+        ENV_NAME=$(jq .system.environment_name /opt/elasticbeanstalk/deploy/configuration/containerconfiguration | sed 's/"//g')
+        TARGET_DIR=/etc/formsg
+
+        echo "Checking if ${TARGET_DIR} exists..."
+        if [ ! -d ${TARGET_DIR} ]; then
+            echo "Creating directory ${TARGET_DIR} ..."
+            mkdir -p ${TARGET_DIR}
+            if [ $? -ne 0 ]; then
+                echo 'ERROR: Directory creation failed!'
+                exit 1
+            fi
+        else
+            echo "Directory ${TARGET_DIR} already exists!"
+        fi
+        echo "Creating config for ${ENV_NAME} in ${AWS_REGION}"
+        aws ssm get-parameter --name "${ENV_NAME}-general" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' > $TARGET_DIR/.env
```

Comment:
is it safe to run this in production when these parameter stores do not exist yet?

Reply:
Indeed it is, but I will create the parameter store ahead of time.

Reply:
make no mistake; this is a migration that we will have to perform carefully - recommend releasing standalone.

Reply:
we could also spin up an identical EB environment, then do a DNS cutover after testing that the deploy works as intended